react/Card/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { PureComponent } from 'react'
+
+import styles from './styles.styl'
+
+export class Card extends PureComponent {
+  render() {
+    const { children } = this.props
+    return <div className={styles['c-card']}>{children}</div>
```

Comment:
I think you should spread props:

```jsx
import cx from 'classnames'

render() {
  const { className, ...rest } = this.props
  return <div className={cx(styles['c-card'], className)} {...rest} />
}
```

So if we need to give DOM props to the component, it's possible.

Reply:
Well spotted, thanks.